pkg/lib/infra/whatsapp/errors.go

```diff
@@ -42,8 +53,8 @@ func (e *WhatsappAPIError) SkipLogging() bool {
 	case 401:
 		return true
 	default:
-		if e.ParsedResponse != nil {
-			if firstErrorCode, ok := e.ParsedResponse.FirstErrorCode(); ok {
+		if e.OnPremisesResponse != nil {
```

Comment:
Also handle `CloudAPIResponse` here? (Although it could be unreachable because cloud api does not return such error)